client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -543,7 +539,7 @@ export class CheckoutThankYouHeader extends PureComponent {
 	getHeaderText() {
 		const { purchases, _n } = this.props;
 
-		if ( this.isBulkDomainTransfer() ) {
+		if ( isBulkDomainTransfer() ) {
```

Comment:
I think you need this

```suggestion
		if ( isBulkDomainTransfer( purchases ) ) {
```

Reply:
Thank you, it should already be fixed 👍